### ISSUE TITLE
getLatex() and setLatex(), with error reporting

### ISSUE THREAD

**Issue by laughinghan** [mathquill/mathquill#455]
TL;DR: new math field methods to replace `.latex()`
- `.getLatex()` returns contents as LaTeX
- `.setLatex('a_n x^n')` parses argument as LaTeX and replaces contents, or returns error

You know what's dumb? The "semi-chainable" `.latex()` method overloaded with getter and setter signatures and a heterogenous return type: the setter signature returns `this` math field for [chaining][]; the getter signature _of the same method_ return a string value.

[chaining]: http://schier.co/blog/2013/11/14/method-chaining-in-javascript.html

Yeah, jQuery does it, e.g. [the `.html()` method][], that's what `.latex()` is based on, but we're not jQuery. jQuery's `.html(x)` setter is basically a wrapper for `.innerHTML = x;`, there's no possible contract jQuery could have with its API consumer that could be violated, so there's no need for jQuery to provide error handling (any problems setting `innerHTML` don't involve jQuery, and would be detected and resolved by the API consumer without involving jQuery).

[the `.html()` method]: http://api.jquery.com/html/

By contrast, our LaTeX setter has a complex (and undocumented :weary:) contract about what constitutes valid LaTeX, but if the contract is violated, the only indication we give our API consumer is to empty the math field, which, uh, isn't the friendliest behavior (#277).

Furthermore, with jQuery you want to chain your traversal and filtering methods with your manipulation methods, and there's many things to manipulate, not just content but also attributes, styling, and event handlers. MathQuill has no traversing or filtering, and all there is to manipulate is content and config.

Now, we could make a backcompat-breaking change to the setter signature of the overloaded `.latex()` method....or we could deprecate it backwards-compatibly and introduce new separate getter and setter methods, where the setter does some actual error reporting.

Note that separate getters and setters are the norm for JS libs providing some kind of editor widget, whether syntax-highlighting code editors or rich text editors:
- [CodeMirror](http://codemirror.net/doc/manual.html#api_content): `doc.getValue()`/`doc.setValue()`
- [ACE](http://ace.c9.io/#howto): `.getValue()`/`.setValue()`
- [Quill Rich Text Editor](http://quilljs.com/docs/api/) (no relation): `.getText()`/`.setText()`, `.getHTML()`/`.setHTML()`, `.getContents()`/`.setContents()`
- even the ~~bloated~~ complex [CKEditor](http://docs.ckeditor.com/#!/guide/dev_howtos_miscellaneous): `.getData()`/`.setData()`
- and even the assumedly sarcastically named [TinyMCE](http://www.tinymce.com/wiki.php/api4:class.tinymce.Editor): `.getContent()`/`.setContent()`

I'm thinking of getting these in and deprecating `.latex()` (have it `console.warn()` or something) for the Beta release, then removing it entirely by 1.0.
